db/migrations/1_create_posts.sql

```diff
@@ -0,0 +1,9 @@
+DROP TABLE IF EXISTS posts;
+
+CREATE TABLE posts (
```

Comment:
Perhaps you could add in NOT NULL constraints to stop someone from posting nothing?